frontend/src/lib/components/ReverseProxyChecker/reverseProxyCheckerLogic.test.ts

```diff
@@ -56,6 +62,9 @@ describe('reverseProxyCheckerLogic', () => {
         useMockedValues(hasReverseProxyValues)
 
         logic.mount()
+
+        await logic.asyncActions.loadHasReverseProxy()
+
         await expectLogic(logic).toFinishAllListeners().toMatchValues({
             hasReverseProxy: true,
         })
```

Comment:
```suggestion
        await expectLogic(logic, () => {
        	logic.actions.loadHasReverseProxy()
        }).toFinishAllListeners().toMatchValues({
            hasReverseProxy: true,
        })
```

iirc you can write the tests like this too